mbf_abstract_nav/src/abstract_controller_execution.cpp

```diff
@@ -72,6 +72,7 @@ AbstractControllerExecution::AbstractControllerExecution(
   private_nh.param("dist_tolerance", dist_tolerance_, 0.1);
   private_nh.param("angle_tolerance", angle_tolerance_, M_PI / 18.0);
   private_nh.param("tf_timeout", tf_timeout_, 1.0);
+  private_nh.param("ignore_check_tolerance", robot_ignore_check_tolerance_, 5.0);
```

Comment:
```suggestion
  private_nh.param("ignore_check_tolerance", ignore_check_tolerance_, 5.0);
```
S: as a general rule try to keep the same name for params and variables